__tests__/app.test.js

```diff
@@ -137,3 +137,59 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200- should return an array of comment objects for a given article id, with each comment object containing the correct properties", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toEqual([
+          {
+            comment_id: 11,
+            body: "Ambidextrous marsupial",
+            article_id: 3,
+            author: "icellusedkars",
+            votes: 0,
+            created_at: "2020-09-19T23:10:00.000Z",
+          },
+          {
+            comment_id: 10,
+            body: "git push origin master",
+            article_id: 3,
+            author: "icellusedkars",
+            votes: 0,
+            created_at: "2020-06-20T07:24:00.000Z",
+          },
+        ]);
+      });
```

Comment:
A solid happy path test, though could be made a bit more flexible by looping through the array using something like `toMatchObject`, or just checking the general shape of the properties on the comments objects.
This could be beneficial because if, for example, you added another property to the comment objects in the future, you would need to rewrite this test.